src/components/AutoComplete/AutoComplete.component.tsx

```diff
@@ -107,6 +110,17 @@ export const AutoComplete = (props: AutoCompleteProps) => {
 		};
 	}, []);
 
+	/**
+	 * If isOpen is true and value is empty, then set the active index to 0 and scroll the list to (0,0)
```

Comment:
```suggestion
	 * If "isOpen" is true and value is empty, then set the active index to 0 and scroll the list to (0,0)
```
we can treat variable name as noun and wrap them in `" "` to signify that